### PR TITLE
updated-backspace-for-power-number

### PR DESCRIPTION
**Pull request by matthewjchin**
Hi there,

I really love your idea for creating a basic calculator. I though it maybe could use a little more innovation by putting in other functions to make it more than just a basic arithmetic calculator. 

Instead of your backspace button on y1, x1, instead it is a button where the number is powered. I used the java.lang.Math API and the pow function that will take in the first number and then power that number to whatever the user inputs as the second number. 

Let me know what you think of this as I plan on hopefully adding more features to make it a scientific calculator. This is just the start and I hope it is a bigger project of yours. 

Best,
Matthew Chin

### CODE REVIEW

**Comment by DanielLyi** [comment]
I also love your idea, can add more features to it

**Comment by HouariZegai** [comment]
Thank you for your contribution, but i found some issues with your pull request:
* why you removed backspace feature from UI?
* Root feature doesn't work correctly
* Added buttons (power, root) not compatible with colored theme
Power feature work without problem,

Actually we made **Standard** calculator only, it's will be a good idea if we add other type of calculators like: scientific, statistics and programmer calculator.

Please, fix this problems and i will be happy merge your pull request,
Thanks again!

**Comment by matthewjchin** [comment]
Hi there, just updated some of the modifications that you preferred. However, I realized that there may be some errors once again but I am trying to get it back to normal as you had hoped. Hope this is well and I hope to make additional changes before submitting new modifications to the base branch.

File: src/com/houarizegai/calculator/Calculator.java [hunk 1]
```diff
@@ -1,13 +1,35 @@
 
-package com.houarizegai.calculator;
+package Calculator.src.com.houarizegai.calculator;
```
Comment (xdvrx1) [inline src/com/houarizegai/calculator/Calculator.java:2]: from `package com.houarizegai.calculator` to `package Calculator.src.com.houarizegai.calculator` 

This does not make any sense.

**Comment by xdvrx1** [comment]
As I am reviewing this, there are good commits, why not just separate them? A good pull request has one focus, so that we can always test whether it will cause problems or not. Thanks.

**Comment by matthewjchin** [comment]
I will have to close this pull request for now. Doesn't seem like there's anything else that can be done to advance the properties of the calculator, but good overall.

**Pull request closed without merging**